src/main/java/com/xatkit/core/recognition/processor/ToxicityPostProcessor.java

```diff
@@ -0,0 +1,93 @@
+package com.xatkit.core.recognition.processor;
+
+import com.xatkit.execution.StateContext;
+import com.xatkit.intent.RecognizedIntent;
+import org.apache.commons.configuration2.Configuration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Analyzes the probability of a comment being toxic with different types of toxicity attributes
+ */
+public class ToxicityPostProcessor implements IntentPostProcessor {
+
+    /**
+     * The PerspectiveAPI Client that will ask the requests to PerspectiveAPI
+     */
+    private PerspectiveapiInterface perspectiveapiClient;
+    private DetoxifyInterface detoxifyClient;
```

Comment:
Related to our discussion on a new abstraction for toxicity clients. We will deal with this too